public/data.json

```diff
@@ -5,206 +5,232 @@
     "linkUrl": "/art/beyond_binary",
     "codeUrl": "https://github.com/rt1301/generative-art/blob/redux_implementation/assets/beyond_binary/sketch.js",
```

Comment:
Could you change all the code URLs to redirect to the main repository and not your fork?

Reply:
Yes I'll do that